Sources/LiveViewNative/Modifiers/View Configuration Modifiers/PreferredColorSchemeModifier.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PreferredColorSchemeModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 4/24/23.
+//
+
+import SwiftUI
+
+/// Overrides the preferred color scheme for the presentation.
+/// (This modifier propagates up to the nearest presentation container
+/// rather than modifying the content directly.)
```

Comment:
Could use a `Note` for better semantics if you want.
```suggestion
/// - Note: This modifier propagates up to the nearest presentation container
/// rather than modifying the content directly.
```